packages/formdescriptioneditors/backend/sirius-components-formdescriptioneditors-graphql/src/main/java/org/eclipse/sirius/components/formdescriptioneditors/graphql/datafetchers/mutation/MutationDeletePageDataFetcher.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2023, 2023 Obeo.
```

Comment:
New files should only have 2023 once in their copyrights

Reply:
I had problems with my copyright profile, it should be ok now, I corrected all the files with this error.